### PR TITLE
Drop Node 6 and 8 support

### PR DESCRIPTION
**Pull request by marcoow**
…as Node 6 is EOL and Node 8 is super old.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 language: node_js
 
 node_js:
-  - "6"
+  - 10
```
Comment (Turbo87) [inline .travis.yml]: if we still support Node 8 then we should probably test on Node 8 😉

Comment (marcoow) [reply .travis.yml]: Yeah, good point - this is essentially copied from simplabs/ember-simple-auth, any idea why we do it like this over there?

* https://github.com/simplabs/ember-simple-auth/blob/master/.travis.yml#L8
* https://github.com/simplabs/ember-simple-auth/blob/master/package.json#L100

Comment (marcoow) [reply .travis.yml]: I dropped support for Node 8 as well - no need to keep that really.

Comment (Turbo87) [reply .travis.yml]: looks like I missed that when I looked at https://github.com/simplabs/ember-simple-auth/pull/1956

**Review by Turbo87 — APPROVED** [review]
fine by me. I think Node 8 will be EOL by the end of the year anyway.

**Pull request merged**